discord_bot.py

```
@@ -0,0 +1,195 @@
+        return
+    date = date or _today()
+    try:
+        state = StoryState()
+        digest = state._load_digest(date)
+        pkt = [p for p in digest.get("story_packets", []) if p.get("id")==story_id]
+        if not pkt:
+            await interaction.response.send_message("Story not found.", ephemeral=True)
+            return
+        outline = generate_outline(pkt[0])
+        await interaction.response.send_message(f"```markdown\n{outline}\n```", ephemeral=True)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)